modules/plugin/imagemosaic/src/test/java/org/geotools/gce/imagemosaic/ImageMosaicRepositoryTest.java

```diff
@@ -259,6 +259,119 @@ public void createFromEmptyStore() throws Exception {
         ds.dispose();
     }
 
+    /**
+     * Test for GEOS-8311 load a mosaic when it doesn't have a projection set.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void createFromEmptyStoreWithNoProjection() throws Exception {
+        // setup mosaic
+        URL storeUrl = TestData.url(this, "rgba");
+        File testDataFolder = new File(storeUrl.toURI());
+        File testDirectory = crsMosaicFolder.newFolder("rgba_no_proj");
+        FileUtils.copyDirectory(testDataFolder, testDirectory);
+        // clean up, other tests might have left the rgba source dir dirty
+        for (File file : testDirectory.listFiles(f -> f.getName().startsWith("rgba"))) {
+            file.delete();
+        }
+        // remove the .prj files
+        for (File file : testDirectory.listFiles(f -> f.getName().endsWith("prj"))) {
+            file.delete();
+        }
+        // signal the intention to use a store name using both datastore.properties and indexer
+        Properties properties = new Properties();
```

Comment:
There is no need to add a datastore and indexer for this test, default behavior should be just fine